fish_prompt.fish

```diff
@@ -753,6 +757,9 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   # Save the last status for later (do this before the `set` calls below)
   set -l last_status $status
 
+  # hg settings
+  set -l theme_hg_ignored_dirs $HOME
```

Comment:
This will override whatever the user sets, right? If we're going to do this, let's change it to…

```
set -q theme_hg_ignored_dirs
  or set -l theme_hg_ignored_dirs $HOME
```

?